docs/editors.md

```diff
@@ -40,6 +40,30 @@ vim.lsp.config('ty', {
 vim.lsp.enable('ty')
 ```
 
+## Zed
+
+ty is included with Zed out of the box (no extension required), although the default primary LSP for Python is basedpyright.
+
+You can enable ty and disable basedpyright by adding this to your `settings.json` file:
+
+```json
+{
+  "languages": {
+    "Python": {
+      "language_servers": [
+        // Disable basedpyright and enable Ty, and otherwise
```

Comment:
I saw that comments inside JSON code blocks were used in ruff's doc, so I guess it's okay, right?